tests/__main__.py

```
@@ -6,6 +6,8 @@
 import coverage
 import coverage.exceptions as _cov_exceptions
 
+from tests._utils.profile import profile
```

Ruff (0.8.2)

`tests._utils.profile.profile` imported but unused

Remove unused import: `tests._utils.profile.profile`

(F401)